testutil.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func RequireDecEqual(t *testing.T, expected, actual Dec, msgAndArgs ...interface{}) {
+	if !expected.IsEqual(actual) {
+		assert.Fail(t, fmt.Sprintf("%s != %s", expected, actual), msgAndArgs...)
+		t.FailNow()
+	}
+}
+
+func RequireAccountEqual(t *testing.T, expected, actual Account) {
```

Comment:
Looks nice. Can't fid any usages of this here, will we be using it in later PRs?

Reply:
Yes